release/src/main/python-release/python_release_automation_utils.sh

```diff
@@ -132,6 +153,31 @@ function get_asc_name() {
 }
 
 
+#######################################
+# Create a new virtualenv and install the SDK
+# Globals:
+#   BEAM_PYTHON_SDK
+# Arguments:
+#   $1 - SDK type: tar, wheel
+#   $2 - python interpreter version: [python2.7, python3.5, ...]
+#######################################
+function install_sdk() {
+  sdk_file=$(get_sdk_name $1)
+  print_separator "Creating new virtualenv with $1 interpreter and installing the SDK from $sdk_file."
+  echo "test place 2"
+  gsutil version -l
+  rm -rf ./temp_virtualenv
+  virtualenv temp_virtualenv -p $2
+  . temp_virtualenv/bin/activate
+  gcloud_version=$(gcloud --version | head -1 | awk '{print $4}')
```

Comment:
What is $4 used here? Seems it's copied from `run_release_candidate_python_mobile_gaming.sh`

Reply:
It is an awk variable, which means split the output by a whitespace separator and print 4-th column - you can play with it in the command line. Since $4 is inside quotes, to pass a function arg in this case one would have to write ${4}.

Reply:
see. Thanks!